### PR TITLE
do not sum lists

### PR DESCRIPTION
**Pull request by yanchenko-igor**
We want to be able to overwrite default configuration.

### CODE REVIEW

File: postgres-appliance/scripts/configure_spilo.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def deep_update(a, b):
                 a[key] = b[key]
         return a
     if type(a) is list and type(b) is list:
-        return a + [i for i in b if i not in a]
+        return a
```
Comment (avaczi) [inline postgres-appliance/scripts/configure_spilo.py:130]: The function description needs to be adjusted accordignly

Comment (Jan-M) [reply postgres-appliance/scripts/configure_spilo.py:130]: I agree

**Comment by avaczi** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**